openmrs-19/src/test/java/org/motechproject/openmrs19/resource/impl/EncounterResourceImplTest.java

```diff
@@ -69,6 +70,22 @@ public void shouldCreateEncounter() throws Exception {
     }
 
     @Test
+    public void shouldCreateEncounterWithProviderList() throws Exception {
```

Comment:
this method doesn't test your changes, please verify if created encounter has a provider and encounterProviders fields